docs/syscall_descriptions_syntax.md

```diff
@@ -42,6 +42,9 @@ rest of the type-options are type-specific:
 "stringnoz": a non-zero-terminated memory buffer (no pointer indirection implied), type-options:
 	either a string value in quotes for constant strings (e.g. "foo" or `deadbeef` for hex literal),
 	or a reference to string flags,
+"glob": glob pattern to match on target files, type-options:
+	a zero-terminated glob pattern string in quotes (e.g. "/sys/" or `/sys/**/*` for glob match),
+	optionally followed by a buffer size (string values will be padded with \x00 to that size)
```

Comment:
This seems to disagree with the tests.
Cases with buffer size are listed in errors.txt.